Formula/why3.rb

```
@@ -0,0 +1,57 @@
+class Why3 < Formula
+  desc "Platform for deductive program verification"
+  homepage "http://why3.lri.fr/"
+  url "https://gforge.inria.fr/frs/download.php/file/34797/why3-0.86.2.tar.gz"
+  sha256 "a7ada8bec9b6717257f2932d8b42e5e3f2dd63800f5e9a3f4004229b0efe3389"
+
+  depends_on "menhir"
+  depends_on "ocaml"
+  depends_on "coq" => :optional
+  depends_on "hevea" => [:build, :optional]
+  depends_on "lablgtk" => :optional
+  depends_on "pkg-config" => :build
+  depends_on "rubber" => :build
+
+  def install
+    system "./configure", "--enable-verbose-make",
+                          "--prefix=#{prefix}",
+                          "--with-lispdir=#{share}/emacs/site-lisp/why3"
```

Hmm. Maybe it silently ignores that argument or something :confused: